src/app.service.ts

```
@@ -1,8 +1,289 @@
+public shippingAddrId: number;
+}
 
 @Injectable()
 export class AppService {
+  constructor(private dataSource: any) {}
   getHello(): string {
     return 'Hello World!';
   }
+
+  calculate(array: number[]): { max: number; sum: number } {
+    let max = -Infinity;
```

Biome (1.9.4)

[error] Use Number.NEGATIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.NEGATIVE_INFINITY instead.


(lint/style/useNumberNamespace)